server/service/src/invoice_line/stock_out_line/update/mod.rs

```diff
@@ -18,7 +18,7 @@ use super::StockOutType;
 #[derive(Clone, Debug, PartialEq, Default)]
 pub struct UpdateStockOutLine {
     pub id: String,
-    pub r#type: Option<StockOutType>,
+    pub r#type: StockOutType,
```

Comment:
I got annoyed at one point that I neglected this field and immediately got failing tests due to the `NoInvoiceType` error being thrown... appears it's not optional so I just made it not optional. I'm not 100% sure the GQL layer provides this 100% of the time, though.

If the argument is we can't trust the frontend to send it, I think that's moot as surely then we can't trust it to send the CORRECT one either.

Reply:
Slightly worried this was done for a reason in the frontend, feels like a risky change to make given there's not much testing time left...

Reply:
I agree 😁